pkg/api/user/platform/pgsql/user.go

```diff
@@ -29,15 +29,14 @@ func (u *User) Create(db orm.DB, usr gorsk.User) (*gorsk.User, error) {
 	var user = new(gorsk.User)
 	err := db.Model(user).Where("lower(username) = ? or lower(email) = ? and deleted_at is null",
 		strings.ToLower(usr.Username), strings.ToLower(usr.Email)).Select()
-
-	if err != nil && err != pg.ErrNoRows {
+	if (err == nil) || (err != nil && err != pg.ErrNoRows) {
```

Comment:
This will return ErrAlreadyExists if err is nil, which seems odd.

Reply:
I don't think it's odd. If err is nil it's means that user with this username or email already exists, so we return ErrAlreadyExists.